### ISSUE TITLE
Link "Applies to",  "Canonical order" in propdef tables

### ISSUE THREAD

**Issue by Ms2ger** [w3c/csswg-drafts#3827]
Followup to https://github.com/w3c/csswg-drafts/issues/1139, https://github.com/w3c/csswg-drafts/issues/1867.

https://github.com/w3c/csswg-drafts/issues/1867#issuecomment-428651127 has the following WG resolution:

> RESOLVED: Cannonical [sic] propdef should go to OM and applies to propdef should go to cascade.

The issue was then closed without implementing the resolution.

CC @tabatkins @fantasai @frivoal @therealglazou @emilio

**Comment by frivoal** [comment]
@tabatkins I guess this is to be fixed by fixing bikeshed, right?

**Comment by Ms2ger** [comment]
I don't know if bikeshed has something to link to at this point.

**Comment by fantasai** [comment]
Probably the most reasonable things are https://www.w3.org/TR/css-cascade-4/#value-stages for applies-to but it's not super clear. We'd probably benefit from porting over https://www.w3.org/TR/CSS2/about.html#property-defs into ... something. The Snapshot maybe.

**Comment by tabatkins** [comment]
Yeah, open an issue on Bikeshed with the things I should be linking to, and it's trivial to fix.

**Comment by frivoal** [comment]
I don't think it should go into the snapshot. Other than this possible addition, the snapshot is not normative, and it is a moving target. Having a mostly informative moving target as a normative dependency doesn't sound great. I think cascade is a fine host for that. Also, the css2.1 text probably needs an update.

**Comment by fantasai** [comment]
Agenda+ because we can't fix this until CSSOM and css-cascade-4 are updated on /TR.

**Comment by svgeesus** [comment]
If CSSOM is going to be published soon, I should send a PR first to [remove the color serialization stuff](https://drafts.csswg.org/cssom-1/#serializing-css-values) and point to [CSS Color 4 serialization](https://www.w3.org/TR/css-color-4/#serializing-color-values) instead. /TR already has this. This will also stop CSSOM linking to CSS Color 3 for the definition of `<color>`
